config/cesm/machines/config_machines.xml

```diff
@@ -747,13 +750,17 @@ This allows using a different mpirun command to launch unit tests
     </module_system>
 
     <environment_variables>
-      <env name="OMP_STACKSIZE">64M</env>
+      <env name="OMP_STACKSIZE">256M</env>
+      <env name="OMP_PLACES">threads</env>
+      <env name="OMP_PROC_BIND">close</env>
       <env name="FI_CXI_RX_MATCH_MODE">hybrid</env>
       <env name="FI_MR_CACHE_MONITOR">memhooks</env>
     </environment_variables>
+   <!--
```

Comment:
Can you document the performance difference that you saw?

Reply:
The performance is degraded in waccmx quit a bit.
```
FX2000 at f19 resolution:
  with the `MPICH_MPIIO_HINTS` setting:
    Model Throughput:         0.19   simulated_years/day 
  without the `MPICH_MPIIO_HINTS` setting:
    Model Throughput:         0.54   simulated_years/day
```

However, I see `MPICH_MPIIO_HINTS` does improve performance of F2000climo.
```
F2000climo at f09 resolution:
  with the `MPICH_MPIIO_HINTS` setting:
    Model Throughput:         9.30   simulated_years/day
  without the `MPICH_MPIIO_HINTS` setting:
    Model Throughput:         7.90   simulated_years/day
```

Reply:
In this case is fx2000 using threads while f2000climo is not?

Reply:
Both of those cases are not using threads (NTHRDS=1)

Reply:
thanks for clarifying

Reply:
Hi Francis and Jim, just want to provide some data points from my threading tests for comparison:
```
F2000climo at f09 resolution, one CPU node on Derecho (128 CPU cores):
1. no threading, use "mpiexec -np 128": 1.349 SYPD
2. no threading, use "mpibind" for 128 MPI tasks: 1.367 SYPD
3. w/ threading (64 MPI tasks and 2 threads per MPI task), use "mpibind": 1.506 SYPD
```
If Francis is observing a performance degradation here, I wound like to understand whether this is due to compset, resolution, threading or other settings, and provide feedback to CISL later if there is an improvement can be done on CISL's end.